src/components/stylebutton.js

```diff
@@ -12,6 +12,11 @@ export default class StyleButton extends React.Component {
     };
   }
 
+  onMouseDown = (e) => {
```

Comment:
No need of this function. You can call onClick directly in the ontoggle function in constructor. Make sure to check if onClick is present or not. Otherwise other people's projects might break. Also you should pass the ev argument to it

Reply:
I added the changes.

Reply:
Any update on this @brijeshb42